base_layer/core/tests/wallet.rs

```diff
@@ -108,264 +108,264 @@ fn create_peer(public_key: CommsPublicKey, net_address: Multiaddr) -> Peer {
     )
 }
 static EMISSION: [u64; 2] = [10, 10];
-#[test]
-fn wallet_base_node_integration_test() {
-    let shutdown = Shutdown::new();
-    let alice_temp_dir = tempdir().unwrap();
-    let bob_temp_dir = tempdir().unwrap();
-    let factories = CryptoFactories::default();
+// #[test]
```

Comment:
What's the reason for this?

Reply:
Removed as it was broken at the time of writing. The merge conflict should fix this.